app/core/src/main/res/values/arrays_account_settings_values.xml

```diff
@@ -18,11 +18,33 @@
         <item>0xFF3949AB</item> <!-- Indigo 600 -->
 
         <item>0xFFE91E63</item> <!-- Pink 500 -->
-        <item>0xFF8E24AA</item> <!-- Purple 600-->
-        <item>0xFF5E35B1</item> <!-- Deep purple 600-->
+        <item>0xFF8E24AA</item> <!-- Purple 600 -->
+        <item>0xFF5E35B1</item> <!-- Deep purple 600 -->
         <item>0xFF455A64</item> <!-- Blue gray 700 -->
     </integer-array>
 
+    <integer-array name="account_colors_dark_theme">
```

Comment:
At least right now we don't have separate account colors for dark mode. So this should probably be named `drawer_account_accent_color_dark_theme` or something similar. And while it's convenient to have the array in this file, it doesn't really belong here. Maybe a separate file named `arrays_drawer.xml`. And add a comment here and in the new file to reference the other array to let developers know the values should be kept in sync.